lib/components/DateInput/DateInput.spec.jsx

```diff
@@ -76,7 +76,7 @@ describe('<DateInput />', () => {
 		fireEvent.focus(container.querySelector('input'));
 
 		fireEvent.change(container.querySelector('input'), {
-			target: { value: 'value from the tests' },
```

Comment:
This was needed (and existing component would have been broken) because setting a date input to "value from tests" is not a date type, so the value never made it through. However, setting it to a date, does actually get set (see snapshot).